docs/pagination.md

```diff
@@ -434,3 +434,31 @@ const Post = new GraphQLObjectType({
 ```
 
 The `limit` can be an integer or a function that returns an integer. This feature is only supported if pagination is supported for you SQL dialect.
+
+### 4. Max and Default Page Size in Pagination
+
+If you want to limit the maximum number of results allowed in a list field,
+you can use the `sqlPageLimit`.
+To set a default page size use `sqlDefaultPageSize`. without a sqlDefaultPageSize, server defaults to a max limit as per the db in use.
```

Comment:
I think it would actually default to no limit right? That's what I experienced locally on Postgres at least, there's just no `LIMIT` clause in the output SQL

Reply:
I wrote this based on the code block [here](https://github.com/join-monster/join-monster/blob/master/src/stringifiers/shared.js#L180): 
```
let limit = ['mariadb', 'mysql', 'oracle'].includes(name)
    ? '18446744073709551615'
    : 'ALL'
```

It seems for Postgres, it is set to MAX, for the others the max limit the DB can handle?
since they do add on a limit clause to the query regardless, it seemed more accurate to state max limit instead of default to no limit.
Should it be changed?

Reply:
Ah ok, that makes sense to me how it is!